scripts/delayed.js

```diff
@@ -1,7 +1,11 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './aem.js';
+import { sampleRUM, loadScript } from './aem.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+
+// Launch script
+loadScript('https://assets.adobedtm.com/2d251f50426c/e52f833be42a/launch-bdb68bbb4cf5-development.min.js', () => {
```

Comment:
If you want to pass an empty second parameter (for attrs) you can just pass null or not pass anything at all so it defaults to undefined `loadScript('...');`

Reply:
First attempt at using the loadScript. Updated.